### ISSUE TITLE
deployment and bootstrapping via PyPI

### ISSUE THREAD

**Issue by dirkpetersen** [FredHutch/scicomp_flask_examples#5]
The dependencies for our templates should be deployable via PyPI so we can bootstrap on any machine that supports pip3. The actual bootstrap command may also be residing in the this package

Examples: 
https://github.com/FredHutch/lf-backup
https://github.com/FredHutch/proxmox-tools 

both of these examples contain a pypi-push.sh script.

**Comment by dtenenba** [comment]
Do we have an account on PyPI? If so maybe you can (securely) share the credentials with me. Otherwise I will just create my own account. 
BTW this seems useful:
http://peterdowns.com/posts/first-time-with-pypi.html

**Comment by dirkpetersen** [comment]
Yes, I will print it for you. I was using exactly that documentation to create the pypi-push.sh script.

Dirk

**Comment by dirkpetersen** [comment]
here is another perhaps interesting bootstrap thingy that just came out 
https://pypi.python.org/pypi/create-flask-app
